api/spec/v1.yaml

```diff
@@ -2112,6 +2118,8 @@ components:
             The decoded `evm.log` event data. We currently support only two types of evm events, ERC20 `Transfer`
             and `Approve`.
             Absent if the event type is not `evm.log`.
+        evm_log_details:
```

Comment:
I think we can name this better.
1) "event" is the more generic, high-level, preferred term; "log" is specific to the serialized structure that an EVM contract emits. So we have `log_name` and `log_params` because they directly describe/reflect that structure, but `event` elsewhere. So, `evm_event_details`. It also matches the struct name better.
2) The above two fields are already "EVM details" of the event, making it a little weird to have a `details` field with only some of the EVM details included there, seemingly arbitrarily. Since the current `EvmEventDetails` is exclusively about the token involved in the event, I suggest `evm_token`, with a description like `Details about the EVM token involved in the event, if any.` (and the struct can be maybe `EvmEventToken`? Less important.).
3) I wonder if an event can involve multiple tokens, e.g. in the case of DEXes. If so, we should perhaps make this field into an array or map. But looking at an example like https://explorer.emerald.oasis.dev/tx/0x9233f42f9cc487abd92586e9320a48efd1cac097df317100cca701f0702ec4bc/logs makes me think that no, there will typically be a separate event for each token involved. So my vote here would be to keep things as-is (= a single token) and if there ever is an event with multiple tokens, then the frontend (or any other client) will have to issue a separate query to retrieve the details of the 2nd, 3rd etc tokens involved.

TLDR: I suggest we rename this to `evm_token`, and adjust the struct name and its description accordingly.

Reply:
Agreed on #1; however I'm leaning slightly more towards `evm_event_details` if/when we start parsing more event types(or contracts, eg DEXes) in the future. In my mind this field is a holding area for any event-related information that isn't included in the event itself. 

OTOH I think `evm_token` would also work for now since we only have token information, and I don't have a clear idea of what other types of contracts we'd eventually support. Maybe DEXes? In any case, I'd prefer to use `evm_event_details` but open to discuss further.

Reply:
Ah, I see you point now. Yeah, valid. It's a judgement call whether `details` is premature generalization (leading to an abstract, harder-to-navigate API) or `token` is us painting ourselves into a corner. I'm still leaning towards `evm_tokens` because
- I can't think of other details either; for better or worse, dapps are still very token-centric
- If other relevant details come up, it's likely they will appear infrequently, and so we'll expect clients to perform separate follow-up queries for those details, rather than perform a slightly expensive JOIN for every `/events` request
- If more details come up that we do want to include in every response, we can still add a new top-level `evm_*` field to contain them.

All of that said, it's not like `_details` is a terrible name, and it's all pretty subjective, so your call, ultimately.

Reply:
Makes sense, I still can't think of any other related non-token info that we'd include. Changed to `evm_tokens`.